### PR TITLE
Feat termdown at 1.11.0

### PR DESCRIPTION
**Pull request by clemensmanert**
###### Motivation for this change
Init termdown 1.11.0

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/termdown/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+click, pyfiglet, dateutil}:
+
+with stdenv.lib;
+
+buildPythonApplication rec {
+
+  name    = "termdown-${version}";
+  version = "1.11.0";
+
+  src = pkgs.fetchFromGitHub {
```
Comment (ndowens) [inline pkgs/applications/misc/termdown/default.nix]: Shouldn't need src = pkgs.fetchFromGitHub, can simply be src = fetchFromGitHub

File: pkgs/applications/misc/termdown/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      rev    = "d1e3504e02ad49013595112cb03fbf175822e58d";
+      sha256 = "1i6fxymg52q95n0cbm4imdxh6yvpj3q57yf7w9z5d9pr35cf1iq5";
+      repo   = "termdown";
+      owner  = "trehn";
+  };
+
+  propagatedBuildInputs = [ dateutil click pyfiglet ];
+
+  meta = {
```
Comment (ndowens) [inline pkgs/applications/misc/termdown/default.nix]: You can do meta = with stdenv.lib; {

That way you don't have to write stdenv.lib in licenses and platforms

**Review by ndowens — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/termdown/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{ pkgs, stdenv, fetchFromGitHub, buildPythonApplication,
```
Comment (grahamc) [inline pkgs/applications/misc/termdown/default.nix]: Please don't pass in `pkgs`, since you've passed in all your dependencies.

**Review by grahamc — CHANGES_REQUESTED** [review]
Nice review, @ndowens. @FRidh can you review for python?

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -25935,6 +25935,8 @@ in {
     };
   };
 
+  termdown = callPackage ../applications/misc/termdown {};
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: Python applications are not called via `python-packages.nix` but directly from `all-packages.nix`.

File: pkgs/applications/misc/termdown/default.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+with stdenv.lib;
+
+buildPythonApplication rec {
+
+  name    = "termdown-${version}";
+  version = "1.11.0";
+
+  src = fetchFromGitHub {
+      rev    = "d1e3504e02ad49013595112cb03fbf175822e58d";
```
Comment (FRidh) [inline pkgs/applications/misc/termdown/default.nix]: indentation

File: pkgs/applications/misc/termdown/default.nix [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+      sha256 = "1i6fxymg52q95n0cbm4imdxh6yvpj3q57yf7w9z5d9pr35cf1iq5";
+      repo   = "termdown";
+      owner  = "trehn";
+  };
+
+  propagatedBuildInputs = [ dateutil click pyfiglet ];
+
+  meta = with stdenv.lib; {
+     description     = "Starts a countdown to or from TIMESPEC";
```
Comment (FRidh) [inline pkgs/applications/misc/termdown/default.nix]: indentation

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by 7c6f434c** [comment]
Fixed the style issues and pushed

**Pull request closed without merging**